### ISSUE TITLE
Plus codes aren’t shown in some countries.

### ISSUE THREAD

**Issue by xemasiv** [google/open-location-code#133]
Hello there,

In Google Maps help page it says that 'Plus codes aren’t shown in some countries.'

* Where do we find this list of countries?
* Is it just some country-based regulation, or some specific flaw / problem in the overall code / implementation?

Curious since I'm looking to support / utilise it in a precision service, thank you.

**Comment by drinckes** [comment]
Right now it's displayed on the dropped pin card in all countries. The short code is only shown for business listings in a few countries (BR, CV, NG, SO, ID, IN, RU), depending how it is received we hope to make it more widely available.

Where are you located, or what country are you interested in?

**Comment by xemasiv** [comment]
In those countries, will it still be possible for users to generate plus codes - and at the same time with Google Maps, resolve those plus codes within their country?

I am from Philippines and I'm currently working on a real-estate platform right now. I think plus codes can greatly help in identifying locations of larger real estate (e.g. establishments, lots) and even small ones, such as the entrance / facing road frontage of a house, apartment building or whatever.

I really find this amazing because, I've been a real-estate sales agent myself and our clients will literally ask for the addresses of our developments - and they will instinctively look it up on google maps.

In a country like ours, most of the time they are just shown a large area or the street - hence, they still ask us things like, where's the building? where's the entrance? This exists whether they're searching for vertical developments, or horizontal ones, which of these houses are you're selling / is available?

Long story short, it's shocking that "H2J5+8WQ, Manila" is more precise and more easier to search than "2215 Pedro Gil Street, Santa Ana, Manila, Metro Manila", which is the address of a condo development I used to sell.

**Comment by bocops** [comment]
@xemasiv Plus codes can be generated for every location on earth. Basically, as long as there's a set of valid latitude/longitude coordinates, there's a matching plus code.

My guess is that every valid plus code will be resolved by Google Maps as well, even if it is located in a country where a code isn't _shown_ by Maps initially. I have no evidence for that, though, so perhaps @drinckes can confirm that?

Also, here's a link to the help page mentioned in the OP. If plus codes are currently shown everywhere, perhaps the statement "Plus codes aren’t shown in some countries." can be removed.

https://support.google.com/maps/answer/7047426?hl=en

**Comment by xemasiv** [comment]
@bocops yup that's the page I was referring too.

And yes you are correct; actually just tested it right now, I was able to generate and resolve plus codes from the above mentioned countries (BR, CP, NG, RU, etc). And using https://plus.codes/map/ I was also able to generate the shortcodes :+1: 

Already clear on this, thank you very much :100:
